.github/workflows/install.yml

```
@@ -120,12 +127,24 @@ jobs:
         with:
           main-branch-name: ${{ steps.get-branch.outputs.MAIN_BRANCH }}
 
+      - name: Prepare feature deployment
+        if: inputs.DEPLOY_FEATURE == 'true'
```

actionlint (1.7.4)

property "deploy_feature" is not defined in object type {main_branch: string; run_merge_queue: bool}

(expression)